Enable pipe crc regional

ljia5 opened the pull request:
Add pipe crc regional size/position registers
settting to enable crc regional.

ljia5 committed:
Enable pipe crc regional

Add pipe crc regional size/position registers
settting to enable crc regional.

Signed-off-by: Jialin <lin.a.jia@intel.com>

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

xyzhao2018 reviewed (commented):
Please add test done part, thanks.

xyzhao2018 reviewed (commented):
Please add test done part, thanks.

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

ljia5 committed:
Add pipe crc and regional properties

Setting the region postion property and retrieve the CRC value
by adding CRTC properties: CHECKSUM_REGION and CHECKSUM_CRC.

Signed-off-by: Jialin <lin.a.jia@intel.com>
